tests/Cms/Site/SiteActionsTest.php

```diff
@@ -88,10 +88,10 @@ public function testSave()
 	public function testUpdate()
 	{
 		$site = $this->site()->update([
-			'copyright' => 2018
+			'copyright' => '2018'
 		]);
 
-		$this->assertSame(2018, $site->copyright()->value());
+		$this->assertSame('2018', $site->copyright()->value());
```

Comment:
Why this has to change? Just curious as it is outside the line of the other changes.

Reply:
See the commit description:

> With process isolation, the behavior of the form classes is different and the value was converted to string. By already passing a string, the test succeeds consistently both with and without process isolation.

Reply:
Don't ask me why the Form classes do that BTW. Wasn't able to track that down, but with the old form classes I also don't think it's really worth investigating. ;)